packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -127,6 +127,7 @@ const MobileDialog = (props: React.PropsWithChildren<TMobileDialog>) => {
                             className={classNames({
                                 [`dc-mobile-dialog__${wrapper_classname}`]: wrapper_classname,
                             })}
+                            data-testid='dt_mobile_dialog_stats'
```

Comment:
could you please discuss with Shayan this approach. As I've seen, there is a card for removing all hardcoded testId's.
I'm not sure why we should do it, but if it has been confirmed, so we should start apply this approach

Reply:
```
fireEvent.click(container.querySelector('.accordion-toggle-arrow'));
const mobile_dialog = document.body.querySelector('.dc-mobile-dialog__accumulators-stats');
```

In our tests we were directly accessing DOM node and was throwing error so I had to change to data-testid for this reason. It did not pass CircleCI and Codecov using this approach. I only used data-testid for when absolutely necessary.

Reply:
I understand , but it was suggested to keep testId not hardcoded, but passed as a prop. could you please catch up with Shayan regarding this approach

Reply:
@henry-deriv We don't need changes in this file. Please use existing data-testid 'dt_mobile_dialog' we already have. Also please check: https://github.com/binary-com/deriv-app/pull/8268/files#r1170888883

Reply:
@henry-deriv we are going to remove hard coded data-testid ( at least for the reusable components) and here you can read about the reason : https://medium.com/@kavianrabbani/the-hidden-danger-of-hardcoded-data-testid-in-testing-8ec9e671955e

so if u are testing a page in which `mobile-dialog` is being rendered more than once, then u need to remove it and instead pass testid as prop in each usage.

Reply:
@shayan-deriv noted. Thanks for your input.